src/app/clients/add/page.tsx

```diff
@@ -0,0 +1,18 @@
+import { Metadata } from "next";
+
+import React from "react";
+import AddClientForm from "@/app/clients/add/formComponents";
+
+const AddClients: () => React.ReactElement = () => {
+    return (
+        <main>
+            <AddClientForm />
+        </main>
+    );
+};
+
+export const metadata: Metadata = {
+    title: "AddClients",
```

Comment:
this _should_ be the Title in the page tab - so can you add a space between the words?